src/DeviceConnectControl.js

```diff
@@ -18,32 +18,31 @@ type DeviceConnectControlProps = {
 
 class DeviceConnectControl extends React.Component<DeviceConnectControlProps, {}> {
     connectionStatusIcon() {
+        const dashIconStyle = {
+            opacity: '50%',
+            width: '2.5rem',
+            height: '2.5rem',
+            position: 'relative',
+            top: '0.1rem'
+        };
```

Comment:
Let's use CSS classes for our styling rather than inline styles. To be consistent with the rest of the code.